src/server/chromium.ts

```diff
@@ -120,10 +110,17 @@ export class Chromium extends BrowserTypeBase {
       throw new Error('Arguments can not specify page to be opened');
     const chromeArguments = [...DEFAULT_ARGS];
     chromeArguments.push(`--user-data-dir=${userDataDir}`);
-    if (this._debugPort !== undefined)
-      chromeArguments.push('--remote-debugging-port=' + this._debugPort);
-    else
-      chromeArguments.push('--remote-debugging-pipe');
+
+    const debugPortStr = getFromENV('PLAYWRIGHT_CHROMIUM_DEBUG_PORT');
```

Comment:
While the protocol generator uses this, it looks like we lack a real test. I'll add one.